sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/MediaStreamingAudioInternal.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callautomation.implementation.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The MediaStreamingAudioInternal model. */
+public final class MediaStreamingAudioInternal {
+
+    /*
+     * The audio data.
+     */
+    @JsonProperty(value = "data")
+    private byte[] audioData;
```

Comment:
last thing I knew is that this is a base64 string and not a byte array.
@Ninika could you please confirm?